etc/httpd/global.conf

```diff
@@ -603,7 +603,7 @@ RewriteRule ^(/(?:\w{2,3}(?:-\w{2})?/)?mobile/android-download(-beta)?\.html)$ /
 RewriteRule ^(/(?:\w{2,3}(?:-\w{2})?/)?projects/calendar(/.*)?)$ /b$1 [PT]
 
 # bug 947890
-RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?firefox/latest/releasenotes(/?)$ /b/$1firefox/latest/releasenotes$2 [PT]
+RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?firefox/(?:latest/)?releasenotes(/?)$ /b/$1firefox/latest/releasenotes$2 [PT]
```

Comment:
I don't think `firefox/releasenotes/` should pass through to `firefox/latest/releasenotes/`. And since it was throwing 500s I assume some other rule is already passing it through to bedrock.